packages/language-server/src/ls-config.ts

```diff
@@ -15,6 +15,8 @@ const defaultLSConfig: LSConfig = {
         definitions: { enable: true },
         findReferences: { enable: true },
         fileReferences: { enable: true },
+        findComponentUsages: { enable: true },
+        findComponentUsagesIgnoresImports: { enable: true },
```

Comment:
It was requested in the referenced issue but I'm against adding such a functionality (see comments) for consistency. It also makes the implementation a little easier and makes your question obsolete I think.

Reply:
I was back and forth on it. I ended up with an option because this overlapped some of the file reference functionality. I don't mind to remove the option if we don't care about overlap.

Reply:
Done